src/binding.cpp

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h>
```

Comment:
I think this would not be necessary, if you add it once in `src/sass_context_wrapper.h`.